packages/tools/pie-icons-vue/test/IconAlcoholLarge.test.js

```diff
@@ -90,7 +89,8 @@ describe('IconAlcoholLarge (Large variant) ::', () => {
             // Assert
             expect(wrapper.html()).toContain('width="32"');
             expect(wrapper.html()).toContain('height="32"');
-            expect(errorMock).toHaveBeenCalledTimes(1);
+            expect(spy).toHaveBeenCalled();
+            spy.mockRestore();
```

Comment:
This approach is also good. If we go down this path we can probably remove the `afterEach` blocks.